pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -766,6 +766,18 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 							"summary": "More than 80% of the rest calls failed in virt-handler for the last 5 minutes",
 						},
 					},
+					{
+						Alert: "KubevirtVmHighMemoryUsage",
+						Expr:  intstr.FromString("sum by(pod, container) ( kube_pod_container_resource_limits_memory_bytes{pod=~'virt-launcher-.*', container='compute'} - on(pod,container) container_memory_working_set_bytes{pod=~'virt-launcher-.*', container='compute'}) < 20971520"),
```

Comment:
@andreyod I believe we should create a recording rule for this calculation.
The advantage of the recording rule is that it allow you to precompute frequently needed or computationally expensive expressions and save their result as a new set of time series.

Reply:
Done